README.md

```diff
@@ -82,11 +83,12 @@ SummaryProcessor accepts the following context in addition to the Processor's co
   truncate: {
     length: 100,  # Documents will be truncated if it exceeds this character count. (Integer)
     omission: '…' # A string added to the end of document when it's truncated. (String, nil)
-  }
+  },
+  hostname: '…' # Used to check whether or not each URL of `href` attributes is external site. (String)
```

Comment:
@yujinakayama I didn't read the sentence "in addition to the Processor's context". Ok, I remove it. 😓